collections/_blogs/2020-07-16-community-bridge-with-layer5.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title:  "Community Bridge 2020"
+date:   2020-07-16 10:30:05 -0530
+image: /assets/images/posts/2020-07-16-community-bridge-with-layer5/communitybridge.jpg
+author: Kanishkar J
+tags: community, open source, community bridge
+permalink: /blog/community-bridge-with-layer5
+---
+<img class = "thumbnail" src = "/assets/images/posts/2020-07-16-community-bridge-with-layer5/cblogo.png">
```

Comment:
Let's make this larger and float it off the the left or right, so that text wraps around it.